common/constants.go

```diff
@@ -21,4 +21,6 @@ const (
 	PoolSize = "pool_size"
 	// ProcessingTime logging details key for time taken to processing instance.
 	ProcessingTime = "proccessing_time"
+	// KafkaTopic logging key for the name a the kafka topic
+	KafkaTopic = "topic"
```

Comment:
This seems to be used only once - consider inlining it. I see the appeal of having them in a central place if you want to update them, but I don't imagine they will be updated much?